### PR TITLE
Project 1:  Mobile version skeleton

### PR DESCRIPTION
**Pull request by yaseerokino**
## Features Added ♻️ 

- Added a header section for navigation
- Added a headline section, showing an introductory message 
- Used exact color scheme as designed in this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- Added hover effect to the menu  navigationbutton on the header section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,49 @@
...
+      integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="css/style.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h4 class="logo">OKINO</h4>
```
Comment (arslanbisharat) [inline index.html:18]: - [ ] The logo should be a link. You can make it a link using `a tag`.

File: index.html [hunk 2]
```diff
@@ -2,9 +2,49 @@
...
+      <button aria-label="bars" type="button" class="icn-btn">
+        <i class="fa fa-bars"></i>
```
Comment (arslanbisharat) [inline index.html:20]: - [ ] Please make sure the hamburger has a `nav tag
`

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  --neutral-400: #505f79;
+  --neutral-0: #fff;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html body {
```
Comment (arslanbisharat) [inline css/style.css:18]: - [ ] The display should be using `flex` methodology.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+header {
+  background-color: var(--neutral-0);
+  padding: 2em;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  color: var(--primary-400);
+}
+
+.headline {
```
Comment (arslanbisharat) [inline css/style.css:67]: - [ ] Make sure this section has the proper background image (decorations are always background images, not `images` tags)

File: index.html [hunk 5]
```diff
@@ -2,9 +2,49 @@
...
+        target="_blank"
+        >LET'S CONNECT</a
+      >
+      <div class="social-icons">
+        <img src="icons/twitter.svg" alt="twitter" class="icons" />
+        <img src="icons/medium.svg" alt="twitter" class="icons" />
+        <img src="icons/Linkedin.svg" alt="linkedin" class="icons" />
+        <img src="icons/github.svg" alt="github" class="icons" />
+        <img src="icons/angel.svg" alt="github" class="icons" />
+      </div>
```
Comment (arslanbisharat) [inline index.html:47]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @yaseerokino, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by yaseerokino** [comment]
Successfully made all the required changes

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: Well resolved.

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html]: Well resolved.

File: index.html (hunk 5 above)
Comment (rloterh) [reply index.html]: Well resolved

File: css/style.css (hunk 4 above)
Comment (rloterh) [reply css/style.css:68]: well resolved

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Yaseer, 

You did great on this project, and for making the changes requested by the previous reviewer!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Highly organized/clean code
- Good ReadMe
- No linter errors


## Required Changes ♻️
- [ ] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
```
[the remark this answers is not in the document]
Comment (yaseerokino) [reply css/style.css:68]: Thank you for the feedback, although I did not set a specific width for the background image

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+  background-color: var(--neutral-0);
+  background-image: url("../images/back.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
```
Comment (yaseerokino) [inline css/style.css:73]: Update my CSS, changed the background-size property to **cover**, and added a background-position of **center**

File: css/style.css (hunk 6 above)
[the remark this answers is not in the document]
Comment (yaseerokino) [reply css/style.css:68]: Updated my CSS, changed the background-size property to **cover**, and added a background-position of **center center**

[the remark this answers is not in the document]
Comment (yaseerokino) [reply css/style.css:68]: Also, I'm not sure if I'm allowed to use media queries at this particular stage of the project

File: index.html [hunk 8]
```diff
@@ -2,9 +2,71 @@
 <html lang="en">
```
Comment (carlos-ssh) [inline index.html:2]: [OPTIONAL]
- ADD TO YOUR .GITIGNORE YOUR .DS_Store files. Keep your project professional.

**Review by carlos-ssh — APPROVED** [review]
Hi @yaseerokino, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (yaseerokino) [reply index.html:2]: I do use one of the generic node .gitignore files, Also should i merge my PR to my main branch

**Pull request merged**